Directory.Build.props

```diff
@@ -43,10 +43,63 @@
     <Error Condition="'$(UnityManagedPath)' == ''" Text="'UnityManagedPath' not defined. Can't find UnityEngine.dll."></Error>
     <Error Condition="!Exists('$(UnityManagedPath)/UnityEngine.dll')" Text="Couldn't find UnityEngine at $(UnityManagedPath)/UnityEngine.dll."></Error>
     <ItemGroup>
+<!--      <Reference Include="UnityEngine">-->
+<!--        <HintPath>$(UnityManagedPath)/UnityEngine.dll</HintPath>-->
```

Comment:
That here needs some cleaning up but the gist of it is: `ManagedPath/UnityEngine.dll` on the managed root contains all the type forwarders to the modules inside of `ManagedPath/UnityEngine/`. This does not seem to work 100% of the time like with LogAssert in the test projs. Instead, we could just reference the individual modules.